File: WebApplication/src/modules/application/models/application-lot-result.dto.ts
import { ApplicationLotResultType } from '../enums/application-lot-result-type.enum';

export class ApplicationLotResultDto {
  id: number;
  type: ApplicationLotResultType;
  attachedFilePath: string;
  note: string;
  certificateNumber: string;
  accessCode: string;
  isSigned: boolean;
}

File: VisaD.Application/Applications/Dtos/ApplicationLotResultDto.cs
using VisaD.Data.Applications.Enums;

namespace VisaD.Application.Applications.Dtos
{
	public class ApplicationLotResultDto
	{
		public int Id { get; set; }
		public ApplicationLotResultType Type { get; set; }
		public string AttachedFilePath { get; set; }
		public string Note { get; set; }
		public string CertificateNumber { get; set; }
		public string AccessCode { get; set; }
		public bool IsSigned { get; set; }
	}
}
